packages/camera/src/components/Capture/hooks.js

```diff
@@ -254,7 +310,7 @@ export function useCheckComplianceAsync({ compliance, inspectionId, sightId: cur
         payload: { id: sightId, status: 'pending', imageId },
       });
 
-      const result = await monk.entity.image.getOne({ inspectionId, imageId });
+      const result = (await monk.entity.image.getOne({ inspectionId, imageId })).axiosResponse;
```

Comment:
Don't return `axiosResponse` because this is not the purpose of the hook. Return the result ans use axiosResponse where you treat the result.